linkerd/linkerd.go

```diff
@@ -700,8 +552,8 @@ func (iClient *Client) applyConfigChange(ctx context.Context, deploymentYAML, na
 		}
 		err := iClient.k8sDynamicClient.Resource(mappingNamespace.Resource).Delete(namespace, deleteOptions)
 		if err != nil {
-			// logrus.Error(fmt.Sprintf("Delete the %s %s failed", dataNamespace.GetObjectKind().GroupVersionKind().Kind, namespace))
-			// return err
+			logrus.Error(fmt.Sprintf("Delete the %s %s failed", dataNamespace.GetObjectKind().GroupVersionKind().Kind, namespace))
```

Comment:
I saw we command out the code, Is that mean that we do not need to return the error and output the logs?

Reply:
@Aisuko that needs to be corrected. Basically yes we do want to delete, but only if it exists.

Reply:
If I understand you in a right way that we want to ignore the `err` right?